src/main/java/core/basesyntax/strategy/operation/impl/BalanceOperationImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.operation.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.strategy.operation.OperationHandler;
+
+public class BalanceOperationImpl implements OperationHandler {
+    @Override
+    public void getResultBalance(String fruitName, int value) {
+        if (Storage.FRUIT_STORAGE.get(fruitName) == null) {
```

Comment:
Move storage interaction higher by realization (in every OperationHandler assessor)